consensus/polybft/polybft.go

```diff
@@ -629,6 +629,16 @@ func (p *Polybft) startConsensusProtocol() {
 		stopSequence func()
 	)
 
+	// check every block time * 1.5, so we don't artificially close a sequence
```

Comment:
This comment is true only for even block times, for instance 2,4,6...

Reply:
This is just a temporary fix taken from gateway-fm. We will abandon this stale sequence checker as soon as we find a better solution.